__tests__/app.test.js

```diff
@@ -68,17 +66,6 @@ describe("GET /api/articles/:article_id", () => {
       .then((res) => {
         expect(res.body.article).toMatchObject({
           article_id: 1,
-          title: "Living in the shadow of a great man",
-          topic: "mitch",
-          author: "butter_bridge",
-          body: "I find this existence challenging",
-          created_at: "2020-07-09T20:11:00.000Z",
-          votes: 100,
-          article_img_url:
-            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-        });
```

Comment:
I'm a little confused by this change - your test now just checks that you're receiving an article back, but not that it is article 1, which is the important bit of this endpoint